### PR TITLE
Add incidents API to swagger

### PR DESCRIPTION
**Pull request by ityankel**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 1]
```diff
@@ -4442,339 +5178,680 @@
...
       },
-      "type": "object",
-      "x-ms-discriminator-value": "Fusion"
+      "required": [
+        "value"
+      ]
     },
-    "GeoLocation": {
-      "description": "The geo-location context attached to the ip entity",
+    "IncidentOwnerInfo": {
```
Comment (ryansbenson) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: Have you talked to anyone about the GDPR/JEDI requirements regarding this data?

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 2]
```diff
@@ -2011,6 +2011,121 @@
...
+        "x-ms-examples": {
+          "Get all relations of an entity.": {
+            "$ref": "./examples/entities/relations/GetAllEntityRelations.json"
+          }
+        },
+        "tags": [
+          "EntityRelations"
+        ],
+        "description": "Gets all relations of an entity.",
+        "operationId": "EntitiesRelations_List",
```
Comment (sanjaiganesh) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: >EntitiesRelations_List [](start = 24, length = 22)

ListByEntities

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 3]
```diff
@@ -2396,83 +2511,681 @@
...
-    },
-    "AATPDataConnectorProperties": {
-      "allOf": [
-        {
-          "$ref": "#/definitions/DataConnectorTenantId"
+        "tags": [
+          "Incidents"
+        ],
+        "description": "Gets all incidents.",
+        "operationId": "Incidents_List",
```
Comment (sanjaiganesh) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: >Incidents_List [](start = 24, length = 14)

ListByWorkspaces

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 4]
```diff
@@ -2396,83 +2511,681 @@
...
+            "schema": {
+              "$ref": "#/definitions/Incident"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/Incident"
+            }
+          },
```
Comment (sanjaiganesh) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: is this long running.. ?
If so , set x-ms-long-running-operation:true
also,  add "x-ms-long-running-operation-options" extension with "final-state-via" property.
``` json
"x-ms-long-running-operation-options": {
  "final-state-via": "location"
}
```

or

``` json
"x-ms-long-running-operation-options": {
  "final-state-via": "azure-async-operation"
}
```

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 5]
```diff
@@ -2396,83 +2511,681 @@
...
+        "x-ms-examples": {
+          "Get all incident relations.": {
+            "$ref": "./examples/incidents/relations/GetAllIncidentRelations.json"
+          }
+        },
+        "tags": [
+          "IncidentRelations"
+        ],
+        "description": "Gets all incident relations.",
+        "operationId": "IncidentRelations_List",
```
Comment (sanjaiganesh) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: >IncidentRelations_List [](start = 24, length = 22)

Listbyincident

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 6]
```diff
@@ -2396,83 +2511,681 @@
...
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Relation"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/Relation"
+            }
```
Comment (sanjaiganesh) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: if long running, set x-ms-long-running:true 
also  add "x-ms-long-running-operation-options" extension with "final-state-via" property.
``` json
"x-ms-long-running-operation-options": {
  "final-state-via": "location"
}
```

or

``` json
"x-ms-long-running-operation-options": {
  "final-state-via": "azure-async-operation"
}
```

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 7]
```diff
@@ -4442,339 +5178,676 @@
...
-        },
-        "enabled": {
-          "description": "Determines whether this alert rule is enabled or disabled.",
-          "type": "boolean"
+          "type": "integer"
         },
-        "lastModifiedUtc": {
-          "description": "The last time that this alert has been modified.",
-          "format": "date-time",
+		"commentsCount": {
```
Comment (sanjaiganesh) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: there are few tab chars.. change them to spaces

**Comment by sanjaiganesh** [comment]
Discussed offline.. they are not long running, as they complete < 2 sec. 
Please address tabs->to->spaces.
Please update operationId for all operations,  in another PR as discussed.

**Review by raych1 — APPROVED** [review]

**Pull request merged**